### ISSUE TITLE
Multipart upload samples

### ISSUE THREAD

**Issue by billyyarosh** [ReactiveX/RxNetty#462]
Curious if there is any support for this. Digging and can't seem to find it. Thanks in advanced. :sos:

**Comment by NiteshKant** [comment]
@keaplogik I haven't tried multipart upload anytime, if you try it out and face issues, I am happy to help :)

**Comment by graemerocher** [comment]
It doesn't seem possible to try it, since RxNetty hides the underlying netty request. And the Netty API for multipart handling requires access to the request object:

https://github.com/netty/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostRequestEncoder.java#L154

So as it stands it isn't possible to create multipart request with RxNetty, at least not with the regular Netty APIs for doing so.

**Comment by billyyarosh** [comment]
@graemerocher I agree that you cannot use the native netty tooling. We have been using the raw netty client today for this and will not have time to implement any APIs around the multipart for RxNetty. We may just provide some utilities for specific use cases like file upload.
